### ISSUE TITLE
Support for windows paths

### ISSUE THREAD

**Issue by lukeXcze** [nuxt-community/router-module#34]
### Version

[v1.3.0](https://github.com/router-module/releases/tag/v1.3.0)

### Reproduction link

[// windows node.js needed](// windows node.js needed)

### Steps to reproduce

Building NUXT project with [`@nuxtjs/router`](https://www.npmjs.com/package/@nuxtjs/router) module on windows node.js
fail by compile error:

This dependency was not found:
outer.js in ./.nuxt/router.js
outer.jsll it, you can run: npm install --save ..

### What is expected ?

in generated `@/.nuxt/router.js`:
```
import { createRouter as customCreateRouter } from '../router.js'
// ..
```.

### What is actually happening?

in generated `@/.nuxt/router.js`:
```
import { createRouter as customCreateRouter } from '..\router.js'
//...
```
windows backslash resolve to escaped \r in windows

### Additional comments?

Windows node returns backshash in relative paths:
https://nodejs.org/api/path.html#path_path_relative_from_to

fixing `@nuxtjs\router\lib\module.js` line  31:
```
      routerFilePath: relative(this.options.buildDir, routerFilePath).replace(/\/+|\\+/g, '/'),
```
...solves the problem
original line:
```
      routerFilePath: relative(this.options.buildDir, routerFilePath).replace(/\/+/g, '/'),
```
...replaces only slash, not backslash

<!--cmty--><!--cmty_prevent_hook-->
<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/router-module/issues/c34">#c34</a>)</em></sub></div>

**Comment by lukeXcze** [comment]
Hi,
node.js native `path.relative()` on non-windows platform returns something like `../router.js`
it is ok, no replace needed i guess

on windows platform node.js `path.relative()` return something like `..\\router.js` (backslash is escaped)
i need replace baskslash sign `\` by slash `/` (node.js can resolve normal slash as directory separator)

regExp of `/\\/g` (matches `\`) needed to replace backslash in windows
replace of two slashes (regExp `/\/\//g` matches `//`) doesn't help.

**Comment by ricardogobbosouza** [comment]
Hi, @lukeXcze 
See if solved in version 1.3.2

**Comment by lukeXcze** [comment]
Hi @ricardogobbosouza 
now it's ok, thank you

regExp `/\/+|\\+/g` fixes possible `..//any//path//router.js` or `..\\any\\path\\router.js` to same `../any/path/router.js`
Question is if `+` quantificators are nesesary - if more [back-]slashes can input replace function, but it is safe and it can spoil nothing
